companies/roundyearfun.json

```diff
@@ -0,0 +1,25 @@
+{
+    "slug": "roundyearfun",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Round Year Fun",
+    "address": "None Listed",
```

Comment:
I think @baltpeter would want this in the DB, does he? :D

Reply:
No, unfortunately we cannot accept companies without an address. Also, the only source is 404ing for me.

But we can keep this open in case someone finds their address in the future.